monai/bundle/scripts.py

```diff
@@ -52,12 +55,61 @@ def _update_args(args: Optional[Union[str, Dict]] = None, ignore_none: bool = Tr
 
 
 def _log_input_summary(tag: str, args: Dict):
+    """
+    Log the arguments of bundle scripts.
+
+    Args:
+        tag: tag to identify the script in the log.
+        args: arguments of the script to log.
+
+    """
```

Comment:
Could you remove the first \n in line 66? The format of the first line doesn't look right after using logger info

Reply:
Sure, removed in the latest commit.

Thanks.